bcdaworker/main.go

```diff
@@ -477,6 +478,14 @@ func updateJobQueueCountCloudwatchMetric() {
 	}
 }
 
+func getSegment(ctx context.Context, name string) newrelic.Segment {
```

Comment:
does this fix this issue?
https://jira.cms.gov/browse/BCDA-3145

Reply:
Yes it does! I didn't know that we had a ticket for this. Will update the PR with more details